api/src/workers/3d-network.js

```diff
@@ -0,0 +1,18 @@
+import { Worker } from 'worker_threads'
+
+export default (workerData) => new Promise((resolve, reject) => {
```

Comment:
I think the code looks all good, but some comments/documentation would be nice!

Reply:
Thank you for the review and feedback! I have added some comments now.